itou/approvals/management/commands/fill_approval_origin_fields.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+from django.core.management.base import BaseCommand
+
+from itou.approvals.models import Approval
+from itou.job_applications.models import JobApplicationWorkflow
+
+
+BATCH_SIZE = 1000
+
+
+class Command(BaseCommand):
+    def handle(self, **options):
+        objects_to_migrate = Approval.objects.filter(
+            origin_sender_kind=None,
+            jobapplication__state=JobApplicationWorkflow.STATE_ACCEPTED,
+        ).distinct()
+        total_objects = objects_to_migrate.count()
+        print(f"Before: {total_objects}")
+
+        batch = []
+        for approval in objects_to_migrate.iterator():
+            origin_job_application = approval.jobapplication_set.accepted().earliest("created_at")
```

Comment:
A voir si il n'y a pas besoin de rajouter le `select_related()` ailleurs quand on appelle `get_origin_kwargs()`
```suggestion
            origin_job_application = approval.jobapplication_set.accepted().select_related("sender_prescriber_organization", "to_company").earliest("created_at")
```